packages/ant-component-mapper/src/files/sub-form.js

```diff
@@ -0,0 +1,42 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Typography, Row, Col } from 'antd';
+
+import { useFormApi } from '@data-driven-forms/react-form-renderer';
+
+const { Title, Paragraph } = Typography;
+const SubForm = ({ fields, title, description, FormSpyProvider: _FormSpyProvider, validate: _validate, ...rest }) => {
+  //rest not used. rest ={type, label, name, options, isRequired}
+  const { renderForm } = useFormApi();
+  return (
+    <Fragment>
+      {title && (
+        <Row>
+          <Col span={24}>
+            <Title level={3}>{title}</Title>
+          </Col>
+        </Row>
+      )}
+      {description && (
+        <Row>
+          <Col span={24}>
+            <Paragraph>{description}</Paragraph>
+          </Col>
+        </Row>
+      )}
+      <Row>
+        <Col span={24}>{renderForm(fields)}</Col>
+      </Row>
+    </Fragment>
+  );
+};
+
+SubForm.propTypes = {
+  fields: PropTypes.oneOfType([PropTypes.object, PropTypes.array]).isRequired,
+  title: PropTypes.string,
```

Comment:
Title, description, labels should be all `PropTypes.node` to allow put there anything.

Strings makes problems, when you are trying to use translations. For example, the default way for translating texts in React.Intl library is using `<FormattedMessage id={} defaultMessage={} />` and that is a element, so users would see proptypes errors.